### PR TITLE
Update Malay locale for v2

### LINKED ISSUE #1524: Convert Malay (ms) to v2
We need Malay support and are willing to help convert it to v2 unless someone more familiar is willing/able to do it.

### PR DESCRIPTION
**Pull request by Zyten**
Fixes #1524

### CODE REVIEW

File: src/locale/ms/_lib/match/index.js [hunk 1]
```diff
@@ -1,47 +1,139 @@
 import buildMatchFn from '../../../_lib/buildMatchFn/index.js'
-import buildParseFn from '../../../_lib/buildParseFn/index.js'
 import buildMatchPatternFn from '../../../_lib/buildMatchPatternFn/index.js'
-import parseDecimal from '../../../_lib/parseDecimal/index.js'
 
-var matchOrdinalNumbersPattern = /^(\d+)(th|st|nd|rd)?/i
+var matchOrdinalNumberPattern = /^ke-(\d+)?/i
+// var matchOrdina	lNumberPattern = /^(\d+)(th|st|nd|rd)?/i
+//var matchOrdinalNumberPattern = /^pertama|(ke-)(\d+)?/i
```
Comment (kossnocorp) [inline src/locale/ms/_lib/match/index.js]: Please remove commented code

Comment (Zyten) [reply src/locale/ms/_lib/match/index.js]: My bad. Removed the comments.

Comment (kossnocorp) [reply src/locale/ms/_lib/match/index.js]: No worries, thank you for addressing this!

Comment (Zyten) [reply src/locale/ms/_lib/match/index.js]: You're welcome!

**Pull request merged**